books.txt

```diff
@@ -0,0 +1 @@
+B | 0 | cooking book | 19-11-2020 | 19-12-2020
```

Comment:
Consider removing this file because other users may not use it (add to .gitignore)